packages/core/src/payment/strategies/amazon-pay-v2/amazon-pay-v2-payment-strategy.ts

```diff
@@ -6,21 +6,22 @@ import { OrderActionCreator, OrderRequestBody } from '../../../order';
 import { OrderFinalizationNotRequiredError } from '../../../order/errors';
 import { PaymentArgumentInvalidError, PaymentMethodCancelledError } from '../../errors';
 import PaymentActionCreator from '../../payment-action-creator';
-import PaymentMethodActionCreator from '../../payment-method-action-creator';
 import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
 import PaymentStrategyActionCreator from '../../payment-strategy-action-creator';
 import PaymentStrategy from '../payment-strategy';
 
 import { AmazonPayV2ChangeActionType, AmazonPayV2PaymentProcessor, AmazonPayV2Placement } from '.';
 
+import { guard } from '../../../../src/common/utility';
+import { CheckoutSettings } from '../../../config/config';
```

Comment:
Can we use this type from the payment-integrations-api?

Reply:
Done. Let me know if I'm wrong.